README.md

```diff
@@ -53,11 +53,11 @@ Find your preferred SDK in [our list of official SDKs](#unleash-sdks) and import
 If you use the docker compose file from the previous step, here's the configuration details you'll need to get going:
 
 - For front-end SDKs, use:
-  - URL: `http://localhost:3000/proxy`
-  - `clientKey`: `proxy-client-key`
+  - URL: `http://localhost:4242/api/frontend/`
+  - `clientKey`: `<frontend key you create>`
```

Comment:
should we have an init key for frontend as well?

Reply:
I think yes. The more people can just copy and paste, the better. I think having them create their own is going to be more confusing.